lib/data/answer.csv

```diff
@@ -0,0 +1,6 @@
+question0,question1,question2,question3,question4,question5,question6,question7,question8,question9
```

Comment:
This `data` folder is not the best place to put cabin sorting stuff. I would put it in its own folder similar to how judging algorithm is done in Alyssa and Aryan's PR this week.  

@mike10911 In order to continue to follow our previous folder structure I was also thinking this can be its own npm package rather than modifying package.json / tsconfig.json / yarn.lock of the current internal tools. If that sounds good maybe in a future ticket we can take all our current internal tools repo stuff (since everything goes toward the UI of the website) and stick it in a 3rd folder just like mono-repo does. This can be a refactoring ticket for a group in the coming couple of weeks. I can write it if you need but criteria would be that there are 3 separate npm packages for the cabin algo, juding algo, and website.
Can you add your thoughts on this?

Reply:
In any case Mike I also think cabin sorting ticket progression can mirror the judging tickets. Maybe a new person or two can use those as an example to enhance this in January once website work no longer needs to be done. It's a good "OPTIONAL" project that might be good for learning & resumes

Reply:
Update: the refactoring I proposed has been rescoped to be a part of the ticket! I will bring the cabin sorting enhancements stuff back up with Mike. Resolving